client/boptest.py

```diff
@@ -0,0 +1,262 @@
+import uuid
+import requests
+import json
+import os
+import boto3
+from minio import Minio
+from requests_toolbelt.multipart.encoder import MultipartEncoder
+from shutil import copyfile
+#import getpass
+from pymongo import MongoClient
+import testcase
+
+
+class Boptest:
+
+    # The url argument is the address of the Boptest server
+    # default should be http://localhost/api
+    def __init__(self, url):
+        if url != "http://localhost/api":
+            print("your specified url is incorrect!")
+            self.url = "http://localhost/api"
+        else:
+            self.url = url
+        
+    # The path argument should be a filesystem path to an fmu
+    # this should be equivalent to uploading a file through 
+    # Boptest UI. See code here 
+    # https://github.com/NREL/alfalfa/blob/develop/web/components/Upload/Upload.js#L127
+    # return value should be a string unique identifier for the model
+    def submit(self, path):
```

Comment:
I'm going to skip review on this method for now. @anyaelena will probably be able to help out.